packages/ember-runtime/lib/mixins/action_handler.js

```diff
@@ -212,38 +212,7 @@ const ActionHandler = Mixin.create({
     }
   },
 
-  willMergeMixin(props) {
-    assert('Specifying `_actions` and `actions` in the same mixin is not supported.', !props.actions || !props._actions);
-
-    if (props._actions) {
-      deprecate(
-        'Specifying actions in `_actions` is deprecated, please use `actions` instead.',
-        false,
-        { id: 'ember-runtime.action-handler-_actions', until: '3.0.0' }
-      );
-
-      props.actions = props._actions;
-      delete props._actions;
-    }
-  }
+  willMergeMixin() {}
```

Comment:
I think we can remove the implementation completely?

Reply:
Will this have issues with https://github.com/emberjs/data/blob/master/addon/-private/system/model/model.js#L1889

Reply:
No, removing this empty function shouldn’t affect that (we aren’t removing the functionality yet)...